src/renderer/atlas/pch.h

```diff
@@ -13,6 +13,7 @@
 #include <span>
 #include <string_view>
 #include <vector>
+#include <cassert>

```

Comment:
Code using assert but did not include the header cassert, rely on the unstable internal include relationship of STL. I think it would be better to fix it.